app/src/main/java/org/mozilla/fenix/library/LibrarySiteItemView.kt

```diff
@@ -86,6 +88,14 @@ class LibrarySiteItemView @JvmOverloads constructor(
         isFocusable = mode != ItemType.SEPARATOR
     }
 
+    open fun showOverflowMenu() {
+        overflowView.showAndEnable()
```

Comment:
Instead of making this an `open` class, could `displayAs(mode)` adjust the overflow view instead? Perhaps we could set `overflowView` to be null when displaying as a separator.